src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/AnsiConsole.cs

```diff
@@ -9,34 +9,19 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer, bool useConsoleColor)
+        private AnsiConsole(TextWriter writer)
         {
             Writer = writer;
-
-            _useConsoleColor = useConsoleColor;
-            if (_useConsoleColor)
-            {
-                OriginalForegroundColor = Console.ForegroundColor;
-            }
         }
 
         private int _boldRecursion;
-        private bool _useConsoleColor;
 
-        public static AnsiConsole GetOutput(bool useConsoleColor)
-        {
-            return new AnsiConsole(Console.Out, useConsoleColor);
-        }
+        public static AnsiConsole Output = new AnsiConsole(Console.Out);
```

Comment:
Oh, I only compared against the latest version. Please change the code so that we don't allow setting the static fields - we don't need it and it is ugly...